### ISSUE TITLE
[BUG] `no-invalid-role` should not report error for dynamic role attribute in template expression

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#334]
**Describe the bug**
The no-invalid-role rule is currently reporting a lint error for the following valid use of a template literal:

```js
html`
  <div role="${this.cond ? 'presentation' : 'group'}"></div>
`;
```

**To Reproduce**
https://html-eslint.org/playground#eyJjb2RlIjoiaHRtbGBcbiAgICAgIDxkaXZcbiAgICAgICAgcm9sZT1cIiR7dGhpcy5jb25kID8gJ3ByZXNlbnRhdGlvbicgOiAnZ3JvdXAnfVwiPjwvZGl2PmAiLCJjb25maWciOnsicnVsZXMiOnsiQGh0bWwtZXNsaW50L25vLWludmFsaWQtcm9sZSI6ImVycm9yIn19LCJsYW5ndWFnZSI6ImphdmFzY3JpcHQifQ==

**Expected behavior**

 No lint error for dynamic role values defined via template expressions

**Additional context**
